### PR TITLE
Sass 5590

### PR DESCRIPTION
**Pull request by SamuelTwumBarima**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME](ADD URL HERE)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [ ]  Have you addressed warnings where appropriate?
- [ ]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

File: test/views/offPayrollWorking/EmployerOffPayrollWorkingViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    override val noText: String = "Na"
+    override val expectedParagraph2: String = "Mae hyn yn golygu:"
+    override val expectedLink: String = "Dysgwch ragor am weithio oddi ar y gyflogres (yn agor tab newydd)"
+    override val expectedButtonText: String = "Yn eich blaen"
+    override val expectedHelpLinkText: String = "Help gyda’r dudalen hon"
+  }
+
+  object ExpectedIndividualEN extends SpecificExpectedResults {
+    override val expectedTitle: String = "Off-payroll working (IR35)"
+    override val expectedRadioHeading: String = "Does your client agree with ABC Digitals decision?"
```
Comment (kannanramakrishnan) [inline test/views/offPayrollWorking/EmployerOffPayrollWorkingViewSpec.scala:89]: Required individual heading

File: app/views/offPayrollWorking/EmployerOffPayrollWorkingView.scala.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    <p class="govuk-body" id="employment-opw-paragraph-1">@messages(s"employment.employerOpw.p1.${if(request.user.isAgent) "agent" else "individual"}")</p>
+    <p class="govuk-body" id="employment-opw-paragraph-2">@messages(s"employment.employerOpw.p2")</p>
+    <ul class="govuk-body govuk-list govuk-list--bullet">
+        <li>@messages(s"employment.employerOpw.b1.${if(request.user.isAgent) "agent" else "individual"}")</li>
+        <li>@messages(s"employment.employerOpw.b2.${if(request.user.isAgent) "agent" else "individual"}")</li>
+    </ul>
+
+    <p class="govuk-body">
+        <a href=https://www.gov.uk/government/publications/gains-on-uk-life-insurance-policies-hs320-self-assessment-helpsheet/hs320-gains-on-uk-life-insurance-policies-2022
```
Comment (kannanramakrishnan) [inline app/views/offPayrollWorking/EmployerOffPayrollWorkingView.scala.html:54]: Href Link is incorrect

File: test/views/offPayrollWorking/EmployerOffPayrollWorkingViewSpec.scala [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    override val expectedHeading: String = "Gweithio oddi ar y gyflogres (IR35)"
+    override val expectedParagraph1: String = "Gwnaeth ABC Digital Ltd roi gwybod i CThEF, drwy gyfryngwr, eich bod yn gyflogai iddynt, ac felly yn destun rheolau oddi ar y gyflogres."
+    override val expectedBullet1: String = "gwnaeth ABC Digital Ltd eich trin fel cyflogai at ddibenion treth"
+    override val expectedBullet2: String = "gwnaeth ABC Digital Ltd ddidynnu Treth Incwm a chyfraniadau Yswiriant Gwladol o’ch ffioedd"
+    override val expectedErrorTitle = s"Gwall: $expectedTitle"
+    override val expectedError = "Dewiswch ‘Iawn’ os ydych yn cytuno â phenderfyniad ABC Digital Ltd"
+  }
+
+  object ExpectedAgentEN extends SpecificExpectedResults {
+    override val expectedTitle: String = "Off-payroll working (IR35)"
```
Comment (kannanramakrishnan) [inline test/views/offPayrollWorking/EmployerOffPayrollWorkingViewSpec.scala:110]: expected Title and heading are common expected

File: test/views/offPayrollWorking/EmployerOffPayrollWorkingViewSpec.scala [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    override val expectedRadioHeading: String = "Does your client agree with ABC Digital’s decision?"
+    override val expectedHeading: String = "Off-payroll working (IR35)"
+    override val expectedParagraph1: String = "ABC Digital Ltd has told HMRC your client works for them via an intermediary and is subject to the off-payroll rules."
+    override val expectedBullet1: String = "treated your client as an employee for tax purposes"
+    override val expectedBullet2: String = "deducted Income Tax and National Insurance contributions from their fees"
+    override val expectedErrorTitle = s"Error: $expectedTitle"
```
Comment (kannanramakrishnan) [inline test/views/offPayrollWorking/EmployerOffPayrollWorkingViewSpec.scala:116]: move this to common expected

**Review by kannanramakrishnan — CHANGES_REQUESTED** [review]
Please update the mentioned comments and also the dependencies.

File: test/views/offPayrollWorking/EmployerOffPayrollWorkingViewSpec.scala (hunk 1 above)
Comment (SamuelTwumBarima) [reply test/views/offPayrollWorking/EmployerOffPayrollWorkingViewSpec.scala:89]: done

File: app/views/offPayrollWorking/EmployerOffPayrollWorkingView.scala.html (hunk 2 above)
Comment (SamuelTwumBarima) [reply app/views/offPayrollWorking/EmployerOffPayrollWorkingView.scala.html:54]: done

**Comment by kannanramakrishnan** [comment]
Coverage for EmployerOffPayrollWorkingController is 83%, which will be addressed part of navigation story SASS-5543 as more focus on this story is to render the view with appropriate english and welsh content for individual page.

**Review by kannanramakrishnan — APPROVED** [review]

**Pull request merged**